### PR TITLE
Don't ignore local packages when watching

### PR DESCRIPTION
**Pull request by fcollonval**

### CODE REVIEW

File: quetz_frontend/backend.py [hunk 1]
```diff
@@ -117,23 +117,25 @@ def index(
         )
 
         path = frontend_dir / "static" / file_name
-        if path.exists() and under_frontend_dir(frontend_dir, path):
+        if path.exists():
```
Comment (hbcarlos) [inline quetz_frontend/backend.py:120]: Are you sure it is a good idea to remove the security check `under_frontend_dir(frontend_dir, path)`?
The reason for this check is not to allow requesting files that are not under the `frontend_dir` directory.

Comment (wolfv) [reply quetz_frontend/backend.py:120]: sorry for merging early @hbcarlos

Comment (fcollonval) [reply quetz_frontend/backend.py:120]: It will always be true as `path = frontend_dir / "static" / file_name` (see line above L119)

Comment (hbcarlos) [reply quetz_frontend/backend.py:120]: Great! Thank you soo much @fcollonval!

**Pull request merged**